spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -1594,6 +1596,10 @@ void dump_spdm_error(const void *buffer, size_t buffer_size)
                spdm_response->header.param1,
                spdm_response->header.param2);
 
+        if (spdm_response->header.param1 == SPDM_ERROR_CODE_LARGE_RESPONSE) {
+            printf(" ErrorCode = LargeResponse ");
+        }
```

Comment:
please define an array to add error code string to the array.

Reply:
Thanks. I have submitted anther patch to dump the error.